ale_linters/verilog/iverilog.vim

```diff
@@ -25,7 +25,7 @@ function! ale_linters#verilog#iverilog#Handle(buffer, lines)
         endif
 
         let l:line = l:match[1] + 0
-        let l:type = l:match[2] ==# 'warning' ? 'W' : 'E'
```

Comment:
In iverilog, there are 2 types of error representation, `error` and `syntax error`.
So, I recommend not to change this statement.